app/controllers/developer_controller.rb

```diff
@@ -1,42 +1,72 @@
 class DeveloperController < ApplicationController
 
-  def show_developer
-
+  def show
     @developer = Developer.where(newuser_id: current_newuser.id)
-
-
-
   end
 
-  def create_developer
-
+  def create
+    @developer = Developer.new
```

Comment:
create method should create a new record i.e. persist it to the database
whereas the new method should only create a new instance of the model